platforms/hyperledger-indy/charts/indy-ledger-txn/values.yaml

```diff
@@ -0,0 +1,103 @@
+metadata:
+  namespace:
+  #uisp
+  name: 
+  #custodian-blockchain-indy
+
+replicas:  
+#1
+
+network:
+  name: 
+  #udisp
+
+organization:
+  name:
+    adminIdentity:
+      name: 
+      #admin_name
+      path: 
+      #admin_path
+    newIdentity:
+      name:
+      #identity_name
+      path: 
+      #identity_path
+      role: 
+      #identity_role
+  
+  
+image:
+  initContainer: 
+    name: 
+    #custodian-indy-node
+    repository: 
+    #alpine:3.9.4
+  indyNode: 
+    name: 
+    #custodian-indy-node
+    repository: 
+    #xxxx.accenture.com/uisp/indy-node-1.9.2:SR1.0.0-SNAPSHOT
+  pullSecret: 
+  #uisp
+
+node:
+  name: 
+  #indy-node
+  INDY_NODE_IP: 
+  #0.0.0.0
+  INDY_CLIENT_IP: 
+  #0.0.0.0
+  ports:
+    indyNodePort: 
+    #15911
+    indyClientPort: 
+    #15912
+
+configmap:
+  domainGenesis: 
+  #
+  poolGenesis: 
```

Comment:
you use this as the pool genesis path inside of the job pod, that's fine but you'll need also the pool genesis config map name